mne/decoding/receptive_field.py

```diff
@@ -17,17 +17,18 @@
 class ReceptiveField(BaseEstimator):
     """Fit a receptive field model.
 
-    This allows you to fit a model using time-lagged input features. For
-    example, a spectro- or spatio-temporal receptive field (STRF).
+    This allows you to fit a forward model (stimulus to brain) using
```

Comment:
Personal feeling but I won't block on it, I prefer "encoding model" not "forward model". 
Although we probably shouldn't call it any particular direction of model, since some people may use this to decode (e.g. brain -> spectrogram).

Reply:
That's fine I'll change it to "encoding" and we can fight over it more with @kingjr's stuff :)

Reply:
Better now, or still debatable?

Reply:
Sounds fine to me.